docs/docs/reference/contextual/derivation-macro.md

```diff
@@ -25,7 +25,7 @@ we need to implement a method `Eq.derived` on the companion object of `Eq` that
 produces a quoted instance for `Eq[T]`. Here is a possible signature,
 
 ```scala
-given derived[T: Type](using qctx: QuoteContext) as Expr[Eq[T]]
+given derived[T: Type](using qctx: Quotes) as Expr[Eq[T]]
```

Comment:
What is the recommended name to replace `qctx`? `qt` or `qs`?

Reply:
The recommendation is not to name it. The question is what should we rename `def qctx(using...)...` to? I would like to try `q` as it would make it short for `q.reflect.Tree`.

Reply:
Or we could call it `quotes` to have a meaningful name.

Reply:
This PR is quite large as it is. It might be better to rename that one in another PR

Reply:
I'd actually be in favour of renaming `qctx` to `q`, and also for creating `qr` as an alias to `q.reflect`.

Reply:
I think `q` is acceptable, given that common usage will make it clear. However, I find `qr` is too obscure.

Reply:
I will use `q` as the standard parameter name (or no name). The contextual `qctx` will be renamed to `quotes` in #10442. #10442 also shows how to define your own alias (such as `qr`) in your project with a single line of code.

Reply:
@liufengyun `qr` will stop being obscure if it will be the standard way to refer to the `.reflect` object. Based on my experience from the doctool codebase, I can tell you that `qctx.reflect` feels very verbose. Any codebase that uses Tasty Inspector seriously will run into the same sort of pain.

Anyway, given that we are migrating to an even more verbose name for `qctx`, I think we will define an alias for `quotes.reflect` in the doctool.

Reply:
IMHO introducing two magic names like `q` and `qr` will be too much for the learner.

I think defining a shorter name like `qr` for `q.reflect` in the user library is reasonable.